core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -28,11 +27,10 @@ import spray.can.Http
 import spray.http.StatusCodes
 import spray.json._
 
-import scala.concurrent.duration._
-import scala.util.{Try, Failure, Success}
-
-import scala.concurrent.ExecutionContext.Implicits.global
 import scala.collection.JavaConversions._
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
I see use of `blocking` on Google HTTP calls, and a separate thread pool for slick, which is good. :)